public/app/plugins/datasource/loki/components/monaco-query-field/MonacoQueryField.tsx

```diff
@@ -103,6 +110,34 @@ const MonacoQueryField = ({ history, onBlur, onRunQuery, initialValue, datasourc
     };
   }, []);
 
+  const setPlaceholder = (monaco: typeof monacoTypes, editor: monacoTypes.editor.IStandaloneCodeEditor) => {
```

Comment:
```suggestion
  const setPlaceholder = (monaco: Monaco, editor: monacoTypes.editor.IStandaloneCodeEditor) => {
```

Reply:
The type here should be `Monaco` as that's what monaco is and what we are passing.